doc/lua_api.txt

```diff
@@ -2298,6 +2299,10 @@ These functions return the leftover itemstack.
     * Returns rating of the connect_to_raillike group corresponding to name
     * If name is not yet the name of a connect_to_raillike group, a new group id
     * is created, with that name
+* `minetest.override_on_place(name, func)`: overwrites the on_place of `name`
+    * if func doesn't return true, the previous on_place isn't executed
+* `minetest.override_on_punch(name, func)`: overwrites the on_punch of `name`
+    * if func doesn't return true, the previous on_punch isn't executed
```

Comment:
`isn't`
Using the long form `is not` like in `minetest.raillike_group(name)` makes it more standardized.